Resources/public/js/views/services/plugins/ez-objectrelationsloadplugin.js

```diff
@@ -25,7 +25,7 @@ YUI.add('ez-objectrelationsloadplugin', function (Y) {
         },
 
         /**
-         * Loads the related contents. Once this is done, it sets the contents in
+         * Loads the related contents. Once this is done, it sets the content structs in
```

Comment:
I'm ok with the suggestion.
Only alternative I see is 1. for 1.0.0 and 2. for 1.1.0.